.github/workflows/tests-e2e-npm.yml

```diff
@@ -0,0 +1,63 @@
+name: NPM Static checks
+on:
+  workflow_call:
+    inputs:
```

Comment:
maybe we shoudl take a `testCmd` input so we know what to run. Otherwise every single use of this will need to edit this file. We can even make it an npm script that just runs docker if that would be better to discourage any sort of injection schenanigans